osu.Desktop.VisualTests/KeyCounterTutorial/Count.cs

```diff
@@ -0,0 +1,123 @@
+﻿using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Textures;
+using OpenTK;
+using OpenTK.Graphics;
+
+namespace osu.Desktop.KeyCounterTutorial
+{
+    internal abstract class Count : Container
+    {
+        private int value;
+        public int FadeDuration => 200;
+        public bool IsCounting { get; set; }
+        public bool IsLit { get; set; }

```

Comment:
you should probably be updating the visual state when the state of IsLit changes, rather than having the update calls after each instance you actually change IsLit's status.